src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -99,7 +96,7 @@
   private final IntTextField retreatAfterXRounds = new IntTextField();
   private final IntTextField retreatAfterXUnitsLeft = new IntTextField();
   private final JPanel resultsPanel = new JPanel();
-  private final JButton calculateButton = new JButton("Pls Wait, Copying Data...");
```

Comment:
We may want to keep the old text. The button is disabled until the gameData is cloned. When gameData size gets very large (WorldAtWar rounds 15+), the gameData clone can take upwards of 3~8s. This is really noticeable delay, so some indicator that the window is not active is nice